packages/serverpod/lib/src/server/feature_decisions.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:serverpod/relic.dart';
+import 'package:serverpod_shared/serverpod_shared.dart';
+
+/// Toggles for enabling and disabling features in the server.
+class FeatureDecisions {
```

Comment:
Maybe `FeatureFlags` is a more common name for this?

Reply:
Yes, but I would argue that this is not a flag but the point in the code making the decision based on a source or "flag".  

The implementation is based on the ideas here (but simplified): https://martinfowler.com/articles/feature-toggles.html